src/components/ClientLibraries/ClientLibraries.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { Text } from "@deriv/ui";
+import {
+  clientLibrary,
+  iconJS,
+  logo,
+  logoAndLink,
+  libraryGoTo,
+  libraryChevron,
+} from "./ClientLibraries.styles";
+
+
+export const ClientLibaries = () => {
+  return (
+    <div className={clientLibrary} data-testid="client-header">
+      <div>
+        <div className={iconJS}>
+          <img src="/img/js-library.svg" />
+        </div>
+        <Text
+          type="heading-2"
+          bold
+          css={{
+            "margin-bottom": "40px",
+            "text-align": "center",
+            "@mobile": { "font-size": "32px" },
+          }}
+          role="heading"
+          aria-level={1}
+        >
+          Comprehensive all-in-one client library
+        </Text>
+        <Text
+          type="subtitle-1"
+          align="center"
+          css={{
+            "font-weight": 400,
+            "@mobile": { "text-align": "left" },
+          }}
+          role="heading"
+          aria-level={4}
+        >
+          Use our powerful, flexible, and free API to build a custom trading{" "}
+          <br />
+          platform - for yourself or for your business.
+        </Text>
+        <div className={logo}>
+          <div className={logoAndLink}>
+            <a
+              href="https://binary-com.github.io/deriv-api/"
+              className={libraryGoTo}
```

Comment:
For classname, I suggest that we follow Deriv-app naming convention to standardise it across repos.

Reply:
Used camelCase convention because the old repo also has the naming convention for css. Will change it